deploy/operables.nix

```
@@ -170,7 +171,6 @@ in
 
       ${wait-for-socket}/bin/wait-for-socket "$CARDANO_NODE_SOCKET_PATH"
 
-      export OTEL_SERVICE_NAME=marlowe-chain-indexer
```

Let's keep the default values for these:

```suggestion
export OTEL_SERVICE_NAME="${OTEL_SERVICE_NAME:-marlowe-chain-indexer}"
```

---

@jhbertra , Would that force the need to use OTEL, or would `OTEL_SERVICE_NAME` be harmless?

---

@jhbertra , Would that force the need to use OTEL, or would `OTEL_SERVICE_NAME` be harmless?